dvc/utils/fs.py

```diff
@@ -35,11 +34,13 @@ def get_inode(path):
     return inode
 
 
-def get_mtime_and_size(path, dvcignore):
+# FIXME: There is no use case of get_mtime_and_size without tree, we should
+# move it to tree
```

Comment:
But only to WorkingTree, right? 🙂

Reply:
yep

Reply:
@pared Please remove this FIXME, since you've created an issue already